app/src/main/java/com/android/streetworkapp/ui/utils/CutsomDialog.kt

```diff
@@ -74,3 +82,16 @@ fun CustomDialog(
         )
   }
 }
+
+enum class DialogType {
+  QUERY, // Have submit / cancel buttons + use OnSubmit / onDismiss.
+  INFO; // Only display the content + use onDismiss.
+
+  fun shouldShowSubmitButton(): Boolean {
+    return this == QUERY
+  }
+
+  fun shouldShowCancelButton(): Boolean {
+    return this == QUERY
+  }
```

Comment:
Just one quick question - what is the purpose of these two functions and why not just use the enum types?

Reply:
I did this because we might want more types of Dialogs in the future, I was thinking ahead and tried to get as much of the logic out of the composable as possible. (If there are 4-5 types it would quickly get convoluted in the Dialog's code)

Reply:
This whole PR is done with scalability in mind, for this reason I think it is better as-is for these functions. As we seem to disagree, I propose to refactor them as `shouldShowButtons()`, which is self-explanatory about its use. Do you agree with this proposition ?

Reply:
Agreed, those are functions that are easy to implement later if necessary, furthermore scalability or not two functions doing the same thing is code duplication so as long as refactor them into one this is a good a change